src/test/ppc64.blacklist

```diff
@@ -0,0 +1,43 @@
+blk_nblock
```

Comment:
please move this file to utils/docker to make it clear it is only for CI

Reply:
Done. I also changed ppc64 to ppc64le, to better reflect the intended arch, and is also the arch on travis.